### ISSUE TITLE
Unread count just for mails in inbox

### ISSUE THREAD

**Issue by sentolga** [Thomas101/wmail#532]
Google's inbox mobile app counts unreads just in inbox. Isn't there any API provides this info for each account ? It would be perfect if you provide unread count option as 'Count just Inbox mails'

Best.

**Comment by pawelantczak** [comment]
Try:
Unread Mode: _"All Unread Messages"_ with _"Scrape unread count directly from UI"_ enabled.

**Comment by sentolga** [comment]
I know it but unfortunately it's not what I'm looking for. Your solution also counts the bundles with unread mails as +1 and also it is counting while expanding any conversation because of your dom watching strategy.

What I am asking is just using gmail inbox API for inbox mails which should have specific endpoint for it. I assume they have such API because Google Inbox App works 'perfect' on indicating total unread inbox mails with badges while yours counting 'extras'.

Best,

**Comment by pawelantczak** [comment]
True.

**Comment by Thomas101** [comment]
Yeah I agree. What @sentolga mentioned is the "default" mode that the Inbox iOS and Android apps use and I think this would probably be the best method to use going forwards. The `Scrape unread count directly from UI` is proving to be more problematic that I'd hoped to be honest.

Stay tuned on this one ;-)

**Comment by sentolga** [comment]
Thomas thanks for your update for unread counts based on APIs. But there is confusing issue ;

I am using 2.3.1 pre-released version. my account's unread count setting is inbox (attached) but WMail is counting unread mail inbox mails and also unread mails in only 'updates bundle' as well. interestingly, WMail doesn't count unread mail in other bundles such as Socail, Purchase tc.

BTW, Inbox iOS App doesn't count it as yours does. You may find the screenshots in this comment. 

Best,

![unread_count setting](https://cloud.githubusercontent.com/assets/6025471/24496334/a167018a-1540-11e7-979a-19d3bf8da1e8.png)
![bundle_unreadcount](https://cloud.githubusercontent.com/assets/6025471/24496335/a16add64-1540-11e7-9714-152e98df5755.png)
